volatility3/framework/layers/intel.py

```diff
@@ -236,17 +263,15 @@ def _translate_entry(self, offset: int) -> Tuple[int, int]:
             if table is None:
                 raise exceptions.PagedInvalidAddressException(
                     self.name,
-                    offset,
+                    page_address,
                     position + 1,
                     entry,
                     "Page Fault at entry " + hex(entry) + " in table " + name,
                 )
 
             # Read the data for the next entry
-            entry_data = table[
-                (index << self._index_shift) : (index << self._index_shift)
-                + self._entry_size
-            ]
+            entry_data_start = index << self._index_shift
+            entry_data = table[entry_data_start : entry_data_start + self._entry_size]
```

Comment:
I guess variable assignment saves time over bit-shifting twice?

Reply:
Not drastically hehe, but it does enhance readability a bit ;)